apps/leo_gateway/src/leo_gateway_s3_api.erl

```diff
@@ -412,6 +412,32 @@ head_bucket(Req, Key, #req_params{access_key_id = AccessKeyId}) ->
              {ok, Req} when Req::cowboy_req:req(),
                             Key::binary(),
                             ReqParams::#req_params{}).
+
+%% @doc GET-ACL Operation on Objects
+%% As we only support bucket level ACL, reply bucket ACL here
+%% Related Issue: leo-project/leofs#490
+get_object(Req, Key, #req_params{is_acl = true,
+                                 bucket_name = Bucket}) ->
+    BeginTime = leo_date:clock(),
+    case leo_gateway_rpc_handler:head(Key) of
+        {ok, #?METADATA{del = 0}} ->
+            case leo_s3_bucket:find_bucket_by_name(Bucket) of
+                {ok, BucketInfo} ->
+                    XML = generate_acl_xml(BucketInfo),
+                    Header = [?SERVER_HEADER,
+                              {?HTTP_HEAD_RESP_CONTENT_TYPE, ?HTTP_CTYPE_XML}],
+                    ?reply_ok(Header, XML, Req);
+                not_found ->
+                    ?reply_not_found([?SERVER_HEADER], Bucket, <<>>, Req);
+                {error, _Cause} ->
+                    ?reply_internal_error([?SERVER_HEADER], Bucket, <<>>, Req)
+            end;
+        {ok, #?METADATA{del = 1}}->
+            ?reply_not_found([?SERVER_HEADER], Key, <<>>, Req);
+        {error, Cause} ->
+            ?reply_fun(Cause, get, Bucket, Key, 0, Req, BeginTime)
```

Comment:
@mocchira I've checked the current implementation. I agree we add `GET-ACL` in the method column (1st column).

### the format of the current access log:

| Method | Bucket | Key                                                 | # of Children | Size  | Timestamp                         | Unixtime         | Status Code   | Latency | Cache Status |
|--------|--------|-----------------------------------------------------|---------------|-------|-----------------------------------|------------------|---------------|---------|--------------|
|[PUT]   | test   | test/leo_commons/test/leo_mnesia_tests.erl          |  0            |  3444 | 2017-03-23 10:45:15.521332 +0900  | 1490233515521363 | 200           | 1       |   |
|[PUT]   | test   | test/leo_commons/test/leo_net_tests.erl             |  0            |  2483 | 2017-03-23 10:45:15.530438 +0900  | 1490233515530476 | 200           | 1       |   |    
|[GET]   | test   | test/leo_commons/.git/logs/refs/heads/develop       |  0            |  195  | 2017-03-23 10:45:15.533404 +0900  | 1490233515533439 | 200           | 1       | miss |
|[PUT]   | test   | test/leo_commons/.git/logs/refs/remotes/origin/HEAD |  0            |  195  | 2017-03-23 10:45:15.541014 +0900  | 1490233515541036 | 200           | 1       |   |
|[GET]   | test   | test/leo_commons/test/leo_misc_tests.erl            |  0            |  3248 | 2017-03-23 10:46:40.94936 +0900   | 1490233600094965 | 200           | 1       | hit:mem-cache |
|[GET]   | test   | test/leo_commons/test/leo_misc_tests.erl            |  0            |  3248 | 2017-03-23 10:47:21.930926 +0900  | 1490233641930954 | 200           | 1       | hit:mem-cache |